experimental/traffic-portal/build/traffic_portal_v2.spec

```diff
@@ -25,7 +25,7 @@ License:  Apache License, Version 2.0
 URL:      https://github.com/apache/trafficcontrol/
 Source:   %{_sourcedir}/traffic-portal-%{traffic_control_version}.tgz
 AutoReqProv: no
-Requires: nodejs >= 2:18.0.0
+Requires: nodejs >= 2:20.11.0
```

Comment:
Why require 20.11.0 or higher? TPv2 will run on version 20.0.0, right?

Reply:
That is the version we tested against (internally)